### ISSUE TITLE
Underscore on its own in a text field looks like -

### ISSUE THREAD

**Issue by ProbablePrime** [Frooxius/NeosPublic#1090]
I was working on some Dynamic Variable Stuff where I use String Join to create a variable name. I forgot from the update notes that `-` wasn't allowed in a variable name and so items were not working.

I changed my separator to `_` BUT when adding a single `_` to a text field or input node without any other characters it is rendered basically the same as `-`. As in an underscore looks like a dash if its on its own. The underscore renders in the middle of the text entry.

I'm not sure if this is under your control but it is proving a little confusing. 

tl;dr Underscore looks like dash when it is on its own within a text field.

**Comment by mralext20** [comment]
I belive this is due to the auto vertical centering of text within text fields, and the height settings of the `-` and `_` being the same.

**Comment by Frooxius** [comment]
This is under my control, it's because by default the text renderers use "Geometric" alignment, which ends up centering the whole thing. I can switch the alignment to Line-Based, but it will need some extra tweaks, because it normally results in the line being laid out too low. I'll have to think of some good solutions for this.
